### PR TITLE
Add helm chart

### PR DESCRIPTION
**Pull request by jgsqware**
Signed-off-by: Julien Garcia Gonzalez <julien@giantswarm.io>

### CODE REVIEW

File: helm/reporting-agent-chart/templates/_helpers.tpl [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{{/* vim: set filetype=mustache: */}}
```
Comment (pipo02mix) [inline helm/reporting-agent-chart/templates/_helpers.tpl]: what is that?

File: helm/reporting-agent-chart/templates/agent/1-rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  - ""
+  resources:
+  - daemonsets
+  - deployments
+  - ingresses
+  - namespaces
+  - pods
+  - replicasets
+  - services
+  - statefulsets
```
Comment (pipo02mix) [inline helm/reporting-agent-chart/templates/agent/1-rbac.yaml]: I would add only `pods` for now. We are not scraping anything else for now

**Review by pipo02mix — APPROVED** [review]
LGTM only couple of minor things

File: helm/reporting-agent-chart/templates/_helpers.tpl (hunk 1 above)
Comment (jgsqware) [reply helm/reporting-agent-chart/templates/_helpers.tpl]: Is to enable filetype in vim, and it use mustache templating. I can remove it

File: helm/reporting-agent-chart/templates/agent/1-rbac.yaml (hunk 2 above)
Comment (jgsqware) [reply helm/reporting-agent-chart/templates/agent/1-rbac.yaml]: but if we add more resources type in the configmap, we need to change the RBAC, 

let me try to do it in parallel. Like the same resources in values are use in script and in RBAC, WDYT?

File: helm/reporting-agent-chart/templates/_helpers.tpl (hunk 1 above)
Comment (pipo02mix) [reply helm/reporting-agent-chart/templates/_helpers.tpl]: no just curious

File: helm/reporting-agent-chart/templates/agent/1-rbac.yaml (hunk 2 above)
Comment (pipo02mix) [reply helm/reporting-agent-chart/templates/agent/1-rbac.yaml]: yeah we will need. But now we are not using it. you can reuse `Values.types` here to we dont have two places. I dont see this is going to change soon

Comment (jgsqware) [reply helm/reporting-agent-chart/templates/agent/1-rbac.yaml]: I've set it it like this already, check the latest code. it's working

**Pull request merged**